nixos/modules/services/web-apps/wordpress.nix

```
@@ -98,53 +99,17 @@ let
           type = types.listOf types.path;
```

```suggestion
          type = types.listOf types.package;
```

Or else accessing `plugin.passthru.wpName` and `plugin.pname` makes no sense. Same for themes.